app/src/main/java/at/bitfire/icsdroid/ui/reusable/ColorPickerDialog.kt

```diff
@@ -0,0 +1,52 @@
+/***************************************************************************************************
+ * Copyright © All Contributors. See LICENSE and AUTHORS in the root directory for details.
+ **************************************************************************************************/
+
+package at.bitfire.icsdroid.ui.reusable
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.res.stringResource
+import at.bitfire.icsdroid.R
+import at.bitfire.icsdroid.calendar.LocalCalendar
+import com.maxkeppeker.sheets.core.models.base.SelectionButton
+import com.maxkeppeker.sheets.core.models.base.rememberUseCaseState
+import com.maxkeppeler.sheets.color.ColorDialog
+import com.maxkeppeler.sheets.color.models.ColorConfig
+import com.maxkeppeler.sheets.color.models.ColorSelection
+import com.maxkeppeler.sheets.color.models.MultipleColors
+import com.maxkeppeler.sheets.color.models.SingleColor
+
+@Composable
+fun ColorPickerDialog(
+    initialColor: Int,
+    onSelectColor: (color: Int) -> Unit,
+    onDialogDismissed: () -> Unit,
+) {
+    val templateColors = MultipleColors.ColorsInt(
```

Comment:
I'd give as default the colors provided in [2014 Material Design color palette](https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors). If we don't want to add all of them, we can just pick the ones labeled as 700, which is the default shade AFAIK.

Reply:
Hmm, not completely happy with the 700s only, but I think that does it for now :)